server/configs/webapps/embedded/config/application.properties

```diff
@@ -67,7 +67,7 @@ server.error.include-message=always
 
 ## We do not recommend deploying to a non-root context path (e.g. http://localhost:8080/labkey)
 ## If you do set this property, do not set a legacyContextPath.
-#context.contextPath=/labkey
+#context.contextPath=@@contextPath@@
```

Comment:
This file (`server/configs/webapps/embedded/config/application.properties`) is the one we bundle with distributions. TeamCity doesn't care about it. Changing this _probably_ won't cause any trouble but folks might have scripts that depend on the current text. Let's leave it as is; not worth the risk IMO.

Reply:
If that change isn't there, the plugin won't uncomment that line and the server application.properties file won't configure the contextPath.  It does this check: 
(DoThenSetup.groovy line 110)
`if (configProperties.containsKey("contextPath") && line.contains("=@@contextPath"))
                    {
                        line = line.replace("#context.", "context.")
                    }`

If I undo this change, I think I'll have to check differently or else we don't configure the server for the run